generic/src/main/scala/slog/generic/auto.scala

```diff
@@ -0,0 +1,17 @@
+package slog.generic
```

Comment:
I have serious doubts about somewhat reinventing the wheel by having custom encoders. I'm not sure the benefit outweigh the costs compared to using directly Circe.

But we would have to discuss this in person with a wider group of stakeholders

Reply:
I am strongly in favor of custom encoders. I think we should be able to provide implicit conversion from `io.circe.Encoder[A]` to `slog.StructureEncoder[A]` satisfying circe users while not being locked into it.

Reply:
i agree with custom format but against implicit conversion from circe - just because you can use something in API doesn't mean you should be able to log it in a structured way, imho - you should make an explicit decision about that

Reply:
The decision to import this implicit conversion would still be up to user. I believe that is explicit enough.